### ISSUE TITLE
Editing a Concept does not edit its Reference in a Collection/Dictionary

### ISSUE THREAD

**Issue by brucemakallan** [OpenConceptLab/oclapi#549]
When a Concept is edited, its corresponding reference in a Collection/Dictionary is not updated.

**Comment by brucemakallan** [comment]
Please note that the ticket, [OCLOMRS-481](https://issues.openmrs.org/browse/OCLOMRS-481) is awaiting the resolution of this issue.

**Comment by paynejd** [comment]
FYI this is by design, as a concept could be referenced by lots of
collections, in addition to the the "openmrs linked dictionary", and it
would be bad practice to automatically update the reference everywhere
without providing dictionary managers the chance to first review and
approve the changes.

I expect that this should be addressed in the "OCL for OpenMRS" code, not
in the OCL-API core. However, for a "linked dictionary" like you are
creating, there is definitely an argument to incorporate this behavior into
the API core.

**Comment by djazayeri** [comment]
Yes, we are specifically talking about the "linked dictionary" scenario. Our proposal is that if I have source "Hospital X Custom Concepts" linked to collection "Hospital X Dictionary" then any edits to concepts in the source should automatically update their references in the collection.

(I edited the issue title to reflect this.)
